postgresql-syntax.cabal

```diff
@@ -47,7 +48,11 @@ library
     parser-combinators >=1.3 && <1.4,
     text >=1 && <3,
     text-builder >=0.6.6.3 && <0.7,
+    template-haskell >= 2.11 && < 2.19,
+    th-compat >= 0.1.3 && < 0.2,
     unordered-containers >=0.2.16 && <0.3
+  ghc-options:        -pgmPcpphs -optP--cpp
+  build-tool-depends: cpphs:cpphs
```

Comment:
`/**/` string concatenation in CPP works different between gcc and clang, hence we force the use of cpphs here